muzzle/src/main/java/io/opentelemetry/javaagent/tooling/muzzle/AgentCachingPoolStrategy.java

```diff
@@ -146,7 +146,7 @@ public AgentTypePool typePool(
     return typePool(classFileLocator, classLoader);
   }
 
-  private TypePool.CacheProvider getCacheProvider(ClassLoader classLoader) {
+  TypePool.CacheProvider getCacheProvider(ClassLoader classLoader) {
```

Comment:
usually we add a comment like `// visible for tests` when method visibility is relaxed only for using it from a test